src/libraries/System.IO.Pipes.AccessControl/pkg/System.IO.Pipes.AccessControl.pkgproj

```diff
@@ -1,14 +1,15 @@
 <Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
   <ItemGroup>
-    <ProjectReference Include="..\src\System.IO.Pipes.AccessControl.csproj">
+    <ProjectReference Include="..\ref\System.IO.Pipes.AccessControl.csproj">
       <SupportedFramework>$(NetCoreAppCurrent)</SupportedFramework>
     </ProjectReference>
-    <HarvestIncludePaths Include="lib/net46;runtimes/win/lib/net46" />
-    <HarvestIncludePaths Include="lib/net461;runtimes/win/lib/net461" />
-    <HarvestIncludePaths Include="lib/netstandard1.3" />
-    <HarvestIncludePaths Include="lib/netstandard2.0" />
+    <HarvestIncludePaths Include="ref/net46;lib/net46;runtimes/win/lib/net46" />
+    <HarvestIncludePaths Include="ref/net461;lib/net461;runtimes/win/lib/net461" />
+    <HarvestIncludePaths Include="ref/netstandard1.3;lib/netstandard1.3" />
+    <HarvestIncludePaths Include="ref/netstandard2.0;lib/netstandard2.0" />
     <HarvestIncludePaths Include="runtimes/win/lib/netcoreapp2.1" />
+    <ProjectReference Include="..\src\System.IO.Pipes.AccessControl.csproj" />
```

Comment:
Same comment as the master PR,  please move this right next to the other P2P

Reply:
fixed